W-14455128-Include-new-note-in-API-develompent-studio-CP (#501)
mulesoft/docs-studio: modules/ROOT/pages/api-development-studio.adoc

You can manage APIs for your Mule application the same way you manage the modules that compose your application. By leveraging Exchange as your unified repository for all your Mule application dependencies, you can monitor the latest versions of your dependencies, ensuring that your application is up-to-date with the latest API requirements.

> Because the implementation of your API can change with each iteration, you must keep track so that your Mule application can be updated to match newer specifications. To help you track these changes, you must treat your API specifications as dependencies of your project. This way, every time the specification design changes, the API publishes a new API version that you can manage from Studio and you can update your application to match the new API version.

Reviewer: ```suggestion
Because the implementation of your API can change with each iteration, you must keep track so that your Mule application can be updated to match newer specifications. To help you track these changes, treat your API specifications as dependencies of your project. This way, every time the specification design changes, the API will publish a new API version that you can manage from Studio. You can update your application to match the new API version.
```

Author: OK! Thanks!

Studio leverages APIkit to scaffold the flow and make it easier for you to start developing a Mule application from an API specification.

> NOTE: Studio does not support scaffolding API specs that reference OAS or JSON schema fragments imported from Exchange as project dependencies. Instead, you can scaffold a spec with such fragments by providing the fragments inline within the spec or through fragment files that you create manually within your project directory. RAML fragments from Exchange are not affected by this limitation.

Reviewer: ```suggestion
NOTE: Studio does not support scaffolding API specs that reference OAS or JSON schema fragments imported from Exchange as project dependencies. Instead, you can scaffold a spec with these fragments by providing the fragments inline within the spec or through fragment files that you create manually within your project directory. RAML fragments from Exchange are not affected by this limitation.
```

Reviewer: The "through" isn't parallel to the "by providing the fragments" so it is awkward.

Maybe more clear:
...or through fragment files that you create manually within your project directory. > or by providing fragment files that you create manually within your project directory.

Author: OK! Thanks!

You can start xref:create-api-specification-studio.adoc[an API specification project in Studio and synchronize it with Design Center]. Studio uses version control system (VCS) capabilities to keep your API specifications synchronized between your Studio workspace and Design Center.

> You can also import an API specification from Design Center as an API specification project with VCS enabled capabilities to edit your API specification using Studio and synchronize your changes with Design Center:

Reviewer: ```suggestion
You can also import an API specification from Design Center as an API specification project with VCS-enabled capabilities to edit your API specification using Studio and synchronize your changes with Design Center:
```

Reviewer: I am confused by this paragraph. Is it the VCS-enabled capabilities that edit the API specification? It might be clearer if you break this up.

Author: OK! Thanks!
